src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -363,3 +370,202 @@ function energy_from_T_and_swe(S::FT, T::FT, parameters) where {FT}
     end
 
 end
+
+"""
+    compat_density(W_i::FT, sliq::FT, ρ::FT, Δt_t::FT, Tsnow::FT, density::Anderson1976) where {FT}
+Returns the new compressed density ρ_new from the input ρ under the compaction model defined
+by Snow17 for a given `Anderson1976` parameterization configuration.
+"""
+function compat_density(
+    W_i::FT,
+    sliq::FT,
+    ρ::FT,
+    Δt_t::FT,
+    Tsnow::FT,
+    density::Anderson1976,
+)::FT where {FT}
+    c5 = (sliq > FT(0)) ? FT(2) : density.c5
+    cx = (ρ > density.ρ_d) ? density.cx : FT(0)
+
+    B =
+        FT(100) *
```

Comment:
100 * a has the same type as a, so no need for FT(100) here. 

Also no need to do sliq > FT(0), can compare directly to zero

Reply:
(can propagate to other functions)

note that 0.01*a, if a is Float32, will be Float64

Reply:
Note to self to check with you about some of the examples of these